src/main/java/com/cupid/jikting/common/resolver/AuthorizationResolver.java

```diff
@@ -26,7 +26,6 @@ public boolean supportsParameter(MethodParameter parameter) {
     @Override
     public Long resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest,
                                 WebDataBinderFactory binderFactory) {
-        HttpServletRequest request = (HttpServletRequest) webRequest.getNativeRequest();
-        return jwtService.extractValidMemberProfileId(request.getHeader("Authorization"));
+        return jwtService.extractMemberProfileId(jwtService.extractAccessToken((HttpServletRequest)webRequest.getNativeRequest()));
```

Comment:
p1. 타입 캐스팅 괄호 뒤에 공백 추가해주세요!

```suggestion
        return jwtService.extractMemberProfileId(jwtService.extractAccessToken((HttpServletRequest) webRequest.getNativeRequest()));
```